frontend/src/functions/getL10n.ts

```diff
@@ -16,16 +16,16 @@ export function getL10n() {
     true,
     /\.ftl$/
   );
-  const RESOURCES: Record<string, FluentResource> = {};
+  const RESOURCES: Record<string, FluentResource[]> = {};
 
   for (const fileName of translationsContext.keys()) {
-    const locale = fileName.substring(
-      "./".length,
-      fileName.length - "/app.ftl".length
-    );
+    // Filenames are formatted as `./<locale>/<module>.ftl`.
+    // Example: ./en/bundle.ftl
+    const locale = fileName.split("/")[1];
 
     if (locale) {
-      RESOURCES[locale] = new FluentResource(translationsContext(fileName));
+      RESOURCES[locale] ??= [];
```

Comment:
comment: never seen anyone actually use this `??= ` - awesome :)

Reply:
Haha, I, too, was very excited about a chance to use it :P